TheraLang.BLL/Services/AuthenticationService.cs

```diff
@@ -71,5 +71,28 @@ public async Task<AuthUser> GetAuthUserAsync()
                 };
             });
         }
+
+        public async Task<AuthUser> TryGetAuthUserAsync()
```

Comment:
it's either not good naming or we should refactor this method to match TRY-GET pattern ->  bool TryGetValue(id, our value) signature
https://ayoungdeveloper.com/post/2017-03-28-using-the-tryget-pattern-in-csharp/